app/docs/markdown/blog.md

```diff
@@ -10,28 +10,30 @@ To give you a taste of what we're working on for the next major version of Docks
   * We're working with partners to promote new ways to run CWL and WDL tools and workflows
 * UI rewrite
   * We're currently migrating our UI from AngularJS to Angular (2), watch for performance improvements and usability improvements in this area
-  
+* Write API Web Service and Client!
```

Comment:
I think you want to create a short blog post to tease people that are interested. 
Then in a longer article, still in the UI, explain how someone would choose between 
a) the UI
b) the calling the REST web interface themselves
c) using the write api service 
depending on what they have handy